### PR TITLE
Fixed item_cash

### PR DESCRIPTION
**Pull request by cracken56**
Added spawnflag and output.

### CODE REVIEW

**Review by cracken56 — COMMENTED** [review]
I fucked up something but I didn't really make any changes other than fixing item_cash. Sorry.

**Comment by RyamosThomas** [comment]
You need to fix the line endings before it will be merged

**Comment by Cracken56** [comment]
Done, sorry.

File: csgo_extended.fgd [hunk 1]
```diff
@@ -2774,18 +2774,6 @@
...
-	[
-		"": "No"
-		1: "Yes"
-	]
-
-	prop_weapon_upgrade_helmet (choices) : "Give Prop Helmet" : "" = 
-	[
-		"": "No"
-		1: "Yes"
-	]
```
Comment (Cracken56) [inline csgo_extended.fgd]: Not sure where this came from?

**Comment by cracken56** [comment]
Should be fixed.

**Pull request merged**